providers/rule.rb

```diff
@@ -38,11 +36,11 @@
       batch cmd do
         code cmd
       end
+
     else
       Chef::Log.info("Firewall rule \"#{name}\" already exists.")
```

Comment:
It would be nice to not only be able to differentiate different protocols, but any different values to make this truly idempotent. Right now if a rule with the same name exists but has different values for anything it isn't updated.